api/http/management.go

```diff
@@ -286,19 +286,19 @@ func (h ManagementController) Playback(c *gin.Context) {
 	//nolint:errcheck
 	go h.websocketWriter(ctx, conn, session, deviceChan, errChan, ioutil.Discard)
 
-	err = h.app.GetSessionRecording(ctx,
-		sessionID,
-		app.NewPlayback(sessionID, deviceChan, sleepMilliseconds))
-	if err != nil {
-		err = errors.Wrap(err, "unable to get the session.")
-		l.Error(err)
-		c.JSON(http.StatusInternalServerError, gin.H{
-			"error": "internal error",
-		})
-		return
+	go func() {
+		err = h.app.GetSessionRecording(ctx,
+			sessionID,
+			app.NewPlayback(sessionID, deviceChan, sleepMilliseconds))
+		if err != nil {
+			err = errors.Wrap(err, "unable to get the session.")
+			errChan <- err
+			return
+		}
+	}()
+	// We need to keep reading in order to keep ping/pong handlers functioning.
+	for ; err == nil; _, _, err = conn.NextReader() {
```

Comment:
I had it in comments somewhere to keep the ping/pong going, but according to all my tests it is not needed, in most cases. but any way. no harm to do it. and I wanted to avoid another go routine. anyways, lets try it.